src/_ert/forward_model_runner/cli.py

```diff
@@ -135,21 +143,71 @@ def main(args):
         ee_cert_path,
         experiment_id,
     )
+    reporter_queue: asyncio.Queue[Message] = asyncio.Queue()
+
+    done_flag = asyncio.Event()
```

Comment:
I guess this can be a normal `bool` variable as it is not really awaited, only servers the while loop

Reply:
I cannot see to change it after it has been passed. I tried it, but tests started failing.